### ISSUE TITLE
Do I need to train the vocabulary myself?

### ISSUE THREAD

**Issue by DoMoCo** [lowsfer/RapidBoW#1]
Thank you for your excellent work.

**Comment by lowsfer** [comment]
Yes, it's best if you train the vocabulary with your own dataset / descriptor. It's very easy with the `buildVocabulary` API. And it's very fast. After that you can serialize , save and reuse it in the future.
